pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -544,12 +735,12 @@ pub mod pallet {
 		/// weight for these operations in the `DispatchResultWithPostInfo`.
 		fn process_outbound_message(
 			sender: T::AccountId,
-			domain: Domain,
```

Comment:
Placeholder (might answer itself further down below): I am curios how or where we determine the destination domain instead now?